pkg/server/servicebroker/types_test.go

```diff
@@ -0,0 +1,33 @@
+package servicebroker
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	csvv1alpha1 "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/clusterserviceversion/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func mockCSV() csvv1alpha1.ClusterServiceVersion {
+	return csvv1alpha1.ClusterServiceVersion{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "my-service.v1.0.0",
+		},
+		Spec: csvv1alpha1.ClusterServiceVersionSpec{
+			Description: "A cool description of this service",
+		},
+	}
+}
+
+func TestCSVToServiceParsesDescription(t *testing.T) {
+	var parseDesc = func(desc string) string {
+		require.Equal(t, mockCSV().Spec.Description, desc)
+
+		return ""
+	}
+
+	_, err := csvToService(mockCSV(), parseDesc)
```

Comment:
if you add some markdown to the CSV description and write an expectation that it doesn't have markdown here, this looks good to me

Reply:
I made `ParseDescription` a function interface that is passed to `csvToService` so we can pass a mock and assert it was called. Testing that Markdown is stripped is the responsibility of the `go-strip-markdown` dependency.